Assets/VRTK/Prefabs/SnapDropZone/VRTK_SnapDropZone.cs

```diff
@@ -104,25 +114,13 @@ public enum SnapTypes
         public event SnapDropZoneEventHandler ObjectUnsnappedFromDropZone;
 
         protected GameObject previousPrefab;
-        protected GameObject highlightContainer;
+        public GameObject highlightContainer { get; private set;}
```

Comment:
If this requires a getter, then use a public method called `GetHighlightContainer()` instead of setting a protected method to public like this.